.github/workflows/ci.yml

```diff
@@ -40,29 +35,22 @@ jobs:
         os: [macOS-latest, windows-2019, ubuntu-latest]
     name: cargo clippy+test
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
           toolchain: stable
           components: clippy
-          profile: minimal
-          override: true
 
       - name: restore cache
         uses: Swatinem/rust-cache@v1
```

Comment:
Could you also bump this one to `v2` while at it please?

Reply:
Done.